handout/a11y/testing/screen-readers.md

```diff
@@ -0,0 +1,35 @@
+# Testing with Screen Readers
+The only way to be certain that your application is usable by screen readers and other assistive technologies is to check for yourself. 
+
+To test with a screen reader, you will need to use a screen reading program and navigate your page using keyboard commands specific to that program. When testing with a screen reader, it is best to use Firefox or Internet Explorer, as these are the most popular browsers for users with assistive technologies.
+
+## Testing on Mac using VoiceOver
+VoiceOver is OSX's built-in screen reader, and therefore often the easiest option for developers. 
+
+VoiceOver can be turned on and off with the keyboard shortcut `CMD + F5` and also through the `System Preferences` menu. In the System Preferences menu, you can also adjust the speed at which VoiceOver will read to you. While testing, it is helpful to set it to a higher speed.
+
+[WebAIM's guide for testing with VoiceOver](http://webaim.org/articles/voiceover/) provides detailed usage instructions.
+
+## Testing on Windows using NVDA
+[NVDA](http://www.nvaccess.org/) is a free (but donation funded) screen reader for Windows platforms. 
+
+Once NVDA has been downloaded and installed, you can start it with the keyboard command `Ctrl + Alt + N`. 
+
+[WebAIM's guide for testing with NVDA](http://webaim.org/articles/nvda/) provides detailed usage instructions.
```

Comment:
None that are free, but it may be worth mentioning JAWS as the most popular. Most devs won't have access however.